source/changelog.txt

```diff
@@ -12,6 +12,42 @@ Release Notes
    :depth: 1
    :class: singlecol
 
+v2.3.3
+------
+
+*Released October 30, 2024*
+
+New features released in this version:
+
+- :issue:`MONGOSH-1326` - Add ``shardedDataDistribution`` to
+  :method:`sh.status`.
+- Add individual evergreen test results with XUnit.
```

Comment:
Not too familiar with how we target release notes but this is an internal / not user facing change so may be worth removing

Reply:
Fixed